server/src/test/java/org/opensearch/index/shard/RemoteIndexShardTests.java

```diff
@@ -629,31 +628,27 @@ public void testShallowCopySnapshotForClosedIndexSuccessful() throws Exception {
         }
     }
 
-    @LockFeatureFlag(MERGED_SEGMENT_WARMER_EXPERIMENTAL_FLAG)
     @Override
     @AwaitsFix(bugUrl = "https://github.com/opensearch-project/OpenSearch/pull/18255")
```

Comment:
Looks like these tests are flaky. Can we do a quick follow up after this PR to understand what is the cause of the failures in this test class?